app/controllers/api/v1/lobbies_controller.rb

```diff
@@ -118,6 +141,19 @@ def notify_answer_count(lobby)
         Pusher.trigger(lobby.code, Lobby::ANSWER_SENT, { answer_count: players.count })
       end
 
+      def count_player_score(lobby, player)
+        lobby.quiz.questions.map do |question|
+          correct_answers = question.answers.to_a.select(&:is_correct)
+          number_of_correct_player_answers = PlayerAnswer.where(player: player, answer: correct_answers).count
+          number_of_player_answers = PlayerAnswer.where(player: player, answer: question.answers).count
+          if number_of_correct_player_answers == number_of_player_answers
```

Comment:
This gives the wrong result if the Player doesn't answer to a question, or doesn't provide all correct answers.

Reply:
[This test should theoretically catch the "Player doesn't answer a question" case, no? But it does not, which means it's somehow wrong or produces unexpected results. Any idea?](https://github.com/e-spres-oh/qwizard-api/pull/24/files/880fd6a0b5c70b0f2786d2089cd0d781747cf631#diff-1d942ed499b0b99ef5cf67f5414818c36279c0867c791f7c7aae59d6f3b3755fR360-R365)

Reply:
```
  let(:question1) { FactoryBot.create(:question, quiz: player1.lobby.quiz) }
  let(:question2) { FactoryBot.create(:question, quiz: player1.lobby.quiz) }
```
are not executed for the linked spec. 😄